packages/toolpad-app/src/server/schema.ts

```diff
@@ -261,6 +261,10 @@ export const applicationSchema = toolpadObjectSchema(
   z.object({
     authorization: z
```

Comment:
would it make sense to split `authorization` and `authentication` at this level?